tests/apm/overview.phpt

```diff
@@ -1,6 +1,7 @@
 --TEST--
 PHPC-349: APM Specification
 --SKIPIF--
+<?php if (PHP_INT_SIZE !== 8) { die('skip Only for 64-bit platform'); } ?>
```

Comment:
Why does this test require a 64-bit platform?

Reply:
The cursor ID in one of the returned structures is 64-bit, and can't be represented in 32-bits :-/

Reply:
Looking at Remi's build log, I see:

```
Fatal error: Uncaught MongoDB\Driver\Exception\InvalidArgumentException: Integer overflow detected on your platform: 917 in /builddir/build/BUILD/php-pecl-mongodb-1.4.0~beta1/NTS/tests/apm/overview.php:17
```

If APM can't work on 32-bit, that may be a more general issue to look into. Outside the scope of this PR, though.